CFC_WebApp/main/googlemaps.py

```diff
@@ -0,0 +1,493 @@
+#!/usr/bin/env python
+
+# Copyright 2009 John Kleint
+#
+# This is free software, licensed under the Lesser Affero General 
+# Public License, available in the accompanying LICENSE.txt file.
+
+
+"""
+An easy-to-use Python wrapper for the Google Maps and Local Search APIs.
+
+* **Geocoding**: convert a postal address to latitude and longitude
```

Comment:
It should be checked in because it is outdated and I changed it.

If you downloaded the original one it wont work.

Josh

On Wed, Mar 11, 2015 at 12:46 PM, shankari <notifications@github.com> wrote:

> In CFC_WebApp/main/googlemaps.py
> <https://github.com/e-mission/e-mission-server/pull/102#discussion_r26248257>
> :
>
> > @@ -0,0 +1,493 @@
> > +#!/usr/bin/env python
> > +
> > +# Copyright 2009 John Kleint
> > +#
> > +# This is free software, licensed under the Lesser Affero General
> > +# Public License, available in the accompanying LICENSE.txt file.
> > +
> > +
> > +"""
> > +An easy-to-use Python wrapper for the Google Maps and Local Search APIs.
> > +
> > +* **Geocoding**: convert a postal address to latitude and longitude
>
> Should this be checked in, or should it be a dependency?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/e-mission/e-mission-server/pull/102/files#r26248257>.
>